topostats/run_topostats.py

```diff
@@ -291,7 +290,7 @@ def process_scan(
                     if key in ["grain_image", "grain_mask", "grain_mask_image"]
                 }
                 grainstats = {}
-                for direction in grains.directions.keys():
+                for direction, _ in grains.directions.items():
```

Comment:
I assume there is a reason for doing so, but why iterate over `items()` instead of `keys()` if only the key is to be used?

Reply:
Yep, its [pylint consider-using-dict-items C0206](https://pylint.pycqa.org/en/latest/user_guide/messages/convention/consider-using-dict-items.html).